x-pack/metricbeat/module/aws/cloudwatch/metadata.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cloudwatch
+
+import (
+	awssdk "github.com/aws/aws-sdk-go-v2/aws"
+
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/beats/v7/x-pack/metricbeat/module/aws/cloudwatch/ec2"
+)
+
+// AWS namespaces
+const (
+	namespaceEC2 = "AWS/EC2"
+)
+
+// addMetadata returns a service to fetch metadata from a config struct.
```

Comment:
nit:
```suggestion
// addMetadata adds metadata to the given events map.
```
Feel free to rephrase!

Reply:
Good call, done! Thanks!